pandas/core/frame.py

```diff
@@ -3353,38 +3353,39 @@ def assign(self, **kwargs):
 
         Examples
         --------
-        >>> df = pd.DataFrame({'A': range(1, 11), 'B': np.random.randn(10)})
+        >>> df = pd.DataFrame({'A': range(1, 11),
+        ...                    'B':np.arange(-1.0, 2.0, 0.3)})
```

Comment:
needs a space between `'B':` and `np.arange`

Reply:
Thanks @jschendel for spotting this! I'll send a fix right away.